contracts/PartyBid.sol

```diff
@@ -86,6 +100,7 @@ contract PartyBid is Party {
         // set PartyBid-specific state variables
         marketWrapper = IMarketWrapper(_marketWrapper);
         auctionId = _auctionId;
+        expiresAt = block.timestamp + _durationInSeconds;
```

Comment:
I see no reason to enforce a non-zero duration here, the front-end should prevent it but if someone really wants to make an "immediately-cancellable party" i don't see any reason thats a problem.